students/Sean_Tasaki/Lesson3/mailroom.py

```diff
@@ -0,0 +1,74 @@
+"""
+Sean Tasaki
+5/11/2018
+Lesson03.mailroom_part_1
+"""
+
+donor_full_list = [['Bob Dylan', 120000.00, 3, 40000.00], ['Leonard Cohen', 600.00, 2, 300.00], ['Soren Kierkegaard', 1000.00, 2, 500.00], ['Italo Calvino', 20000.00, 4, 5000.00], ['Florence Feist', 300.00, 3, 100.00]]
+donor_name_list = []
+for x in donor_full_list:
+	donor_name_list.append(x[0])
+
+
+def main_menu():
+	response = input("Enter 1-3 from the following options: (1) Send a Thank You, (2) Create a Report, (3) Quit\n >")
+	if response == '1':
+		thank_you()
+	elif response == '2':
+		create_report()
+	elif response =='3':
+		exit()
+	else:
+		print("Invalid data. Enter 1-3 from the following options: (1) Send a Thank You, (2) Create a Report, (3) Quit\n> ")
+		main_menu()
+
+def thank_you():
+	name = input("Enter the first and last name of the donor or enter 'list' to see a list of previous donor names or enter Q to exit to main menu\n> ")
+	if name.lower() == 'list':
+		print(donor_name_list)
+		thank_you()
+	elif name.upper() == 'Q':
+		main_menu()
+	elif name.title() in donor_name_list:
+		for x in donor_full_list:
```

Comment:
it is better to name your loop var something meaningful like `for donor in donor_full_list`

`x` isn't very clear or readable when reading the code

common exception is using one letter vars in list comprehensions